presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -264,6 +265,18 @@ public void setTimestamp(int parameterIndex, Timestamp x)
         }
     }
 
+    @Override
+    public void setTimestamp(int parameterIndex, Timestamp x, Calendar cal)
+            throws SQLException
+    {
+        checkOpen();
+        if (x == null || cal == null) {
+            setTimestamp(parameterIndex, x);
+            return;
+        }
+        setParameter(parameterIndex, formatLiteral("TIMESTAMP", TIMESTAMP_FORMATTER.withZone(DateTimeZone.forTimeZone(cal.getTimeZone())).print(x.getTime())));
```

Comment:
Maybe wrap this as
```java
setParameter(parameterIndex, formatLiteral("TIMESTAMP", TIMESTAMP_FORMATTER
        .withZone(DateTimeZone.forTimeZone(cal.getTimeZone())).print(x.getTime())));
```

Reply:
this is admittedly long, but fits in our line len limit..
i will extract variable both here & in existing function